Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -169,7 +187,7 @@
 				warning('PostLabelingDelay mismatch with the GE-DICOM value in Inversion time.');
```

Comment:
`GE-DICOM value in Inversion time` -> `GE DICOM Inversion time value` right?
Now it could suggest that Inversion time contains multiple values, or multiple vendors, or whatever.

Reply:
Fixed in 43d03f7b